addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java

```diff
@@ -206,6 +249,77 @@ public void onDeviceStateChanged(ThingUID bridge, Device device) {
 		}
 	}
 
+	private void refreshActualCheck(HeatingThermostat device) {
+		DateFormat dateFormat = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
+
+		if (device.getActualTempLastUpdated() == null) {
+			Calendar t = Calendar.getInstance();
+			t.add(Calendar.MINUTE, REFRESH_ACTUAL_MIN_RATE * -1);
+			device.setActualTempLastUpdated(t.getTime());
+			logger.info("Actual date reset for {} {} ({}) id: {}", device.getType(), device.getName(),
+					device.getSerialNumber(), getThing().getUID());
+		}
+		long timediff = Calendar.getInstance().getTime().getTime() - device.getActualTempLastUpdated().getTime();
+		if (timediff > (refreshActualRate * 1000 * 60)) {
+			if (!refreshingActuals) {
+
+				logger.debug("Actual needs updating for {} {} ({}) id: {}", device.getType(), device.getName(),
+						device.getSerialNumber(), getThing().getUID());
+
+				originalSetTemp = device.getTemperatureSetpoint();
+				originalMode = device.getMode();
+
+				if (originalMode == ThermostatModeType.MANUAL || originalMode == ThermostatModeType.AUTOMATIC) {
+					BigDecimal temporaryTemp = ((DecimalType) originalSetTemp).toBigDecimal().add(
+							BigDecimal.valueOf(0.5));
+					logger.debug("Actuals Refresh: Setting Temp {}", temporaryTemp);
+					handleCommand(new ChannelUID(getThing().getUID(), CHANNEL_SETTEMP), (Command) new DecimalType(
```

Comment:
do you really mean handleCommand here? This method is supposed to actively send a new temperature setpoint to the device, but here you seem to only want to refresh the locally stored value?